roles/cs.aws-rds/defaults/main.yml

```diff
@@ -9,4 +9,15 @@ aws_rds_subnet_group_name: "{{ mageops_app_name }}"
 aws_rds_subnet_group_subnets: "{{ aws_vpc_subnet_ids }}"
 aws_rds_security_groups:
   - "{{ aws_security_group_rds_id }}"
+aws_rds_param_group_name: "{{ mageops_app_name }}"
+aws_rds_param_group_desc: "MageOps parameter group"
+aws_rds_db_engine_version: 5.7
+aws_rds_param_group_engine: "mysql{{ aws_rds_db_engine_version }}"
 
+# Force 1 on 1-2G instances.
+# Force 1-8 on bigger instances.
+# Value needs to be force because of the described bug.
+aws_rds_innodb_buffer_pool_instances: 1
+
+#2 * (NumCPUs + NumDisks)
+aws_rds_innodb_thread_concurrency: 2 
```

Comment:
because both above parameters depends on instance, they should be in `all.yml` instead of role defaults

Reply:
Moved